tests/test_mcp.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
+import logging
+import subprocess
+import time
+from typing import Iterator
+
+import pytest
+
+import pixeltable as pxt
+
+from .utils import skip_test_if_no_client, skip_test_if_not_installed
+
+_logger = logging.getLogger('pixeltable')
+
+
+class TestMcp:
+    def test_mcp_server(self, reset_db: None, init_mcp_server: None) -> None:
+        skip_test_if_not_installed('mcp')
+
+        udfs = asyncio.run(pxt.mcp_udfs('http://localhost:8000/mcp'))
```

Comment:
As discussed, let's make that synchronous.

Reply:
Done.